.github/workflows/ci.yml

```diff
@@ -38,6 +38,7 @@ jobs:
               - 'App/Client/**'
             server:
               - 'App/Server/**'
+              - 'Test/**'
```

Comment:
We don't have anything specific to lint regarding this right now, so we can just remove it from here.